.github/workflows/release.yaml

```diff
@@ -0,0 +1,102 @@
+name: Release Operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          ref: 'main'
+
+      - name: Set up Kubectl
+        uses: azure/setup-kubectl@v1
+
+      - name: Create and Push Tag
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+          git tag ${{ github.event.inputs.version }}
+          git push origin ${{ github.event.inputs.version }}
+
+      - name: Build Operator Image
+        run: |
+          IMG=eda-server-operator:${{ github.event.inputs.version }} make docker-build
+          docker tag eda-server-operator:${{ github.event.inputs.version }} eda-server-operator:latest
+
+      - name: Push Operator Image
+        uses: redhat-actions/push-to-registry@v2.1.1
+        with:
+          image: eda-server-operator
+          tags: ${{ github.event.inputs.version }}
+          registry: quay.io/ansible/
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
```

Comment:
I think this will work nicely, we will just have to make sure that the GHA that builds and pushes the `main` branch image has completed before running the release GHA, which shouldn't really be a problem.  

@jon-nfc I will make the changes you suggested in the [other PR](https://github.com/ansible/eda-server-operator/pull/161) for multi-arch builds as well.

Reply:
Good pickup I didn't even consider a possible collision of not being ready. Although TBH the builds are pretty fast.

further context on the reasoning of why i chose this method is under heading 'Additional info' in ansible/awx-operator/pull/1681